test/test_configuration.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+#include "RtDB2Configuration.h"
+
+void print_comm_settings(RtDB2Configuration& config) {
+    CommunicationSettings cs = config.get_communication_settings();
+    std::cout << "\tmultiCastIP = " << cs.multiCastIP << std::endl;
+    std::cout << "\tfrequency   = " << cs.frequency << std::endl;
+    std::cout << "\tport        = " << cs.port << std::endl;
+    std::cout << "\tcompression = " << cs.compression << std::endl;
+    std::cout << "\tloopback    = " << cs.loopback << std::endl;
+    std::cout << "\tsend        = " << cs.send << std::endl;
+}
+
+int main(int argc, char **argv)
+{
+    std::cout << "=== Default configuration ===" << std::endl;
+    RtDB2Configuration config1;
+    print_comm_settings(config1);
+    std::cout << config1 << std::endl;
+
+    std::cout << "=== Minimal configuration ===" << std::endl;
+    RtDB2Configuration config2;
+    config2.parse_configuration("config/minimal_configuration.xml");
```

Comment:
Maybe we can make this tool just take the provided file name by argument, and add some wrapper test script which feeds it all test/config* files? Then we can easier extend, for instance I could just put in the Falcons config xml and the test would automatically cover it.

Reply:
Partially true. It is also needed to provide the expected output. I will update the pull request.

Reply:
I have made an update. You can now provide any file to test as argument to the test_configuration binary.